index.php

```diff
@@ -7,16 +7,23 @@
   can allow a malicious client to retrieve data about your server and hammer
   a provider throught your rss-bridge instance.
 */
+error_reporting(0);
```

Comment:
We should let the default value as authority if we don't activate debug mode

Reply:
> Actually this line is already present in index.php

Guilty as charged :smile: 

> I can simply remove that line if that's ok with you.

Please do so, there is no reason to keep it (hopefully :pray:)

Reply:
After further research, actually, you just moved the line in 937ea492716b9251211a2a0f73f3afd1ac871505.
It was introduced way before as per 927b04dfef6c363674cfd8e727cd1c34d5485d68 :wink:

The only benefit from this line is that a user will never need PHP error messages... I guess. And someone wanting to debug will enable debug mode anyway.